include/benchmark/benchmark.h

```diff
@@ -438,9 +438,12 @@ enum AggregationReportMode
     : unsigned
 #else
 #endif
-{ ARM_Unspecified,  // The mode has not been manually specified
-  ARM_Default,      // The mode is user-specified as default.
-  ARM_ReportAggregatesOnly };
+{ ARM_Unspecified,           // The mode has not been manually specified
+  ARM_Default,               // The mode is user-specified as default.
+  ARM_ReportAggregatesOnly,  // All reporters only display aggregates.
```

Comment:
does the PR get even simpler if these are renamed to:

ARM_FileReportAggregatesOnly,
ARM_DisplayReportAggregatesOnly,

and then the existing flag sets them both to true, and the new flag sets only the Display one to true?

Reply:
Hmm, kinda.
It gets a bit messier due to the command-line flags.